nntrainer/compiler/tflite_interpreter.cpp

```diff
@@ -172,19 +175,31 @@ class TfOpIdxMap {
 
     for (auto &op_node : nodes) {
       update_opcode(op_node->getOpType());
-      update_variables(op_node->getInputs(), false);
+      /**
+       * Q) Why only add graph input?
+       *
+       * A) nntrainer tensors exist in both directions between nodes (the output
+       * tensor of the "FROM" node, the input of the "TO" node). But in tflite
+       * only one tensor is needed between nodes. Therefore, basically, no
+       * inputs are considered, only graph input that doesn't have "FROM" node
+       * is considered.
+       **/
```

Comment:
Thanks for your clear explanation for better understanding. But, it's too long for the comments.
Could you shorten it into 2-3 lines?
```
The tflite is needed only one tensor between nodes, (cause)....
```